tests/ui-toml/macro_metavars_in_unsafe/default/test.stderr

```diff
@@ -0,0 +1,187 @@
+error: this unsafe block in a macro expands metavariables
```

Comment:
The message doesn't read fluently to me. What do you think about:
```suggestion
error: this unsafe block expands metavariables
```
Or maybe better
```suggestion
error: this macro expands metavariables in an unsafe block
```

Reply:
:+1: I agree, the second suggestion sounds good